src/engine/controls/cuecontrol.cpp

```diff
@@ -22,19 +23,39 @@ static const double CUE_MODE_NUMARK = 3.0;
 static const double CUE_MODE_MIXXX_NO_BLINK = 4.0;
 static const double CUE_MODE_CUP = 5.0;
 
+constexpr double kNoColorControlValue = -1;
+
+namespace {
+
+// Helper function to convert doubles into RgbColor instances (or nullopt)
```

Comment:
how does a single number turn into an RGB value? this needs more explanation

Reply:
This was the reason why I initially wanted to refer to the term *control value* instead of the agnostic term *double*. Unfortunately all our control values are defined implicitly without any type safety.

I agree that a short remark how color control values are interpreted could help. Not here in the implementation, but at the definition of the conversion functions.